Tasks/ServiceFabricUpdateAppVersions/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -6,11 +6,45 @@
   "loc.input.label.applicationPackagePath": "Application Package",
   "loc.input.help.applicationPackagePath": "Path to the application package. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) and wildcards can be used in the path.",
   "loc.input.label.versionSuffix": "Version Suffix",
-  "loc.input.help.versionSuffix": "Value to append to the version values in the manifest files.",
+  "loc.input.help.versionSuffix": "Value to append to the version values in the manifest files. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) and PowerShell expressions can be used.\n\nExample PowerShell expression: .($([DateTimeOffset]::UtcNow.ToString('u').Replace(':', '.')))",
```

Comment:
Good point about using the log command to achieve the same thing I was after. I'll remove this on the off-chance we move away from PowerShell in the future.